controller/array_action/messages.py

```diff
@@ -11,7 +11,7 @@
 
 PoolDoesNotMatchCapabilities_message = "Pool : {0} does not match the following capabilities : {1} . error : {2}"
 
-StorageClassCapabilityNotSupported_message = "Storage class capability is not supported : {0} "
+SpaceEfficiencyNotSupported_message = "space_efficiency is not supported : {0} "
```

Comment:
I would avoid `_` in the user-facing text

Reply:
done